src/backend/controllers/marketController.js

```diff
@@ -442,15 +445,20 @@ exports.marketTransaction = async (req: Request, res: Response) => {
                     }
                 });
 
-                // Remove the sale from database
-                await MarketSale.deleteOne({ _id: saleId }, (err: Error) => {
-                    if (err) {
-                        console.error(err.message);
-                        return res
-                            .status(500)
-                            .json({ msg: 'Could not delete market sale.' });
-                    }
-                });
+                // Remove the sale from database if it does not belong to master account
+                if (sellerId !== String(MASTER_ID)) {
+                    console.log('Sale belongs to a user. Deleting.');
+                    await MarketSale.deleteOne({ _id: saleId }, (err: Error) => {
+                        if (err) {
+                            console.error(err.message);
+                            return res
+                                .status(500)
+                                .json({ msg: 'Could not delete market sale.' });
+                        }
+                    });
+                } else {
+                    console.log('Sale belongs to master account. Not deleting.')
```

Comment:
I had these here for notification purposes between user and master listings, but let me know if I should remove them.

Reply:
That should be fine. As long as it isn't an oversight, I'm cool with it